runtime/src/chpl-comm.c

```diff
@@ -41,8 +41,10 @@
 #include <string.h>
 #include <unistd.h>
 
-int32_t chpl_nodeID = -1;
-int32_t chpl_numNodes = -1;
+int32_t   chpl_nodeID = -1;
+int32_t   chpl_numNodes = -1;
+int32_t   chpl_numLocalesOnNode = 1;
+chpl_bool chpl_oversubscribed = false;
```

Comment:
I worry that some future code may try to use this before it's initialized. Can we add a wrapper function to query oversubscription and have it error if it hasn't been set up yet?

Reply:
I thought the defaults seemed reasonable, but a wrapper function is easily done.

Reply:
As long as we set `chpl_oversubscribed` in `chpl_comm_init()` we should be safe, because that's before even memory layer init, much less tasking layer init. But for paranoia-style coding yeah, a query wrapper wouldn't hurt. I don't think it needs to have a backing `pthread_once()` setter, though.  😁